sample-graphs/update.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -eu
+
+sourcecred_module="$(dirname "$0")""/../sourcecred"
+(cd "${sourcecred_module}"; yarn && yarn backend)
```

Comment:
I’m not sure what the failure looks like if the submodule isn’t
initialized. If you like, you could add a “try running this command”
message for those unfamiliar with submodules (which is many people).

Reply:
Great catch! It indeed gives a fairly inscruable error (package.json not found) when run on a fresh clone. I've modified the script to print a more helpful message, per your suggestion.